projboard/migrations/0006_test_subject_data.py

```diff
@@ -14,7 +14,13 @@ def generate_subject_data(apps, schema_editor):
 
         test_data = [
             'Sport',
-            'Math'
+            'Math',
+            'Software',
+            'Medicine and health',
+            'Business',
+            'Education',
+            'Food',
+            'Hobbies',
```

Comment:
You can add a default one => "Other"

Reply:
There is auto option of "other" by django. it's added by default when we create the form

Reply:
which file and which line does this ?

Reply:
take a look at create-article page...

Reply:
found it